Gemfile

```diff
@@ -111,3 +111,5 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
+gem "webmock", "~> 3.6"
```

Comment:
Is this for testing only? If so, add it within `group :test`.

Reply:
Yes, thanks!